homeassistant/components/plex/__init__.py

```diff
@@ -153,6 +154,11 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         plexapi.exceptions.BadRequest,
         plexapi.exceptions.NotFound,
     ) as error:
+        if INVALID_TOKEN_MESSAGE in str(error):
+            raise ConfigEntryAuthFailed(
```

Comment:
The provided tests do not seem to pick up this case?

Reply:
Oops, you're totally right. A passing test != a good test 😄